.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: Tests
+
+on: [push, pull_request]
```

Comment:
> Also, might want to look into adjusting the build triggers, right now it's on any push/PR.

I don't think TravisCI is/was any more restrictive, so let's start more permissive and can revisit if necessary.